packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -107,6 +107,21 @@ export class LocalSigningManager implements SigningManager {
     return new LocalSigningManager(args.accounts);
   }
 
+  /**
+   * Create an instance of the Local Signing Manager and populates it with a newly generated Account
+   */
+  public static async generate(): Promise<{
```

Comment:
I think we should take out this method and leave `create` as the only way to instantiate accounts.

To make a pre-populated signing manager we could instead add an optional argument specifying how many accounts to generate. 

I think optional params are more scalable than making new methods.

Reply:
I think I see why you introduced this now.

When accounts are made we need to return mnemonics or else they won't be able to recover them.

In that case I think we should expose `createAccount` as a static method on the class, and make the user call that + pass them in like any other account.

Reply:
I agree with the second comment. Also, what's the benefit of returning the keyring pair? Shouldn't just the mnemonic be enough?